### DISCUSSION TITLE
Very interesting new RTMFP in c++ project developed by an ex Adobe developer that...

### DISCUSSION THREAD

**Discussion by ROBERT-MCDOWELL** [ruffle-rs/ruffle#15209] [General]
... can inspire ruffle developers to implement RTMFP and more....
https://github.com/zenomt/rtmfp-cpp

**Comment by al-sabr** [comment]
How about this?

https://github.com/CUB3D/rust-rtmfp

**Reply by al-sabr** [reply]
I ain't a Rust dev but if that repo is fully working you wouldn't need any WASI or WASM modules.

Better keep things Rust only.

Kind Regards

**Reply by fayakun-it-consulting** [reply]
Any feedback?
